core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -95,7 +96,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     AllStatsSignedPercentFromObject("+[amount]% yield from every [tileFilter]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
     StatPercentFromReligionFollowers("[amount]% [stat] from every follower, up to [amount]%", UniqueTarget.FollowerBelief),
     BonusStatsFromCityStates("[amount]% [stat] from City-States", UniqueTarget.Global),
-    StatPercentBonusCities("[amount]% [stat] [cityFilter]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
+    NullifiesStat("Nullifies [stat] [cityFilter]", UniqueTarget.Global),
+    NullifiesGrwoth("Nullifies Growth [cityFilter]", UniqueTarget.Global),
```

Comment:
Nobody saw this typo? Sounds cool though.

Reply:
…I already had four comments, and figured it'd be easy to change anyway. 🤷

Reply:
I'll make a follow-up PR adressing the points of these comments later today :)